viewer/src/datamodels/cad/CadManager.ts

```diff
@@ -35,7 +35,9 @@ export class CadManager<TModelIdentifier> {
   private readonly _subscription: Subscription = new Subscription();
 
   private _needsRedraw: boolean = false;
+
   private readonly _markNeedsRedrawBound = this.markNeedsRedraw.bind(this);
+  private readonly _materialsChangedListener = this.handleMaterialsChanged.bind(this);
```

Comment:
Yes - I think that page is a bit outdated though, the binds are type safe. The reason I do it like this is that `() => this.something()` creates a new capture every time, meaning that the following code doesn't work

```
someone.addEventListener('someEvent', () => this.handleSomeEvent());
someone.removeEventListener('someEvent', () => this.handleSomeEvent());
```

This will register an event handler, but *never* unregister it (!).